medusa/storage/s3_rgw.py

```diff
@@ -33,7 +33,7 @@ def connect_storage(self):
         driver = S3RGWStorageDriver(
             host=self.config.host,
             port=self.config.port,
-            region="default",
+            region="default" if self.config.region.lower() in ('0', 'false') else self.config.region,
```

Comment:
You can actually set the default [here in config.py](https://github.com/thelastpickle/cassandra-medusa/pull/113/files#diff-3feb4fc181a6e98475e3a0899f71011fR84).

It'll get overridden by any value that would be set in the config file.
This way, you can just use `region=self.config.region` here.

@arodrime, I guess we can reuse this `region` setting once we upgrade libcloud and use the new AWS S3 settings to select a region, right?

Reply:
We bumped libcloud dependency here already: https://github.com/thelastpickle/cassandra-medusa/issues/17. This was supposed to bring support for distinct regions. Are you speaking about this?

Other than that I'm not too familiar with using medusa in AWS, I did not play with it myself yet.

Reply:
Thanks for reply, i make some tests and it's ok. My cloud provider is OVH openstack.